validators/activity_pub_signed_requests_validator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ActivityPubSignedRequestsValidator
+  def initialize(opts = {})
+    @opts = opts
+  end
+
+  def valid_value?(val)
+    return true if val == "f"
```

Comment:
Given this is used on list settings, when would the value of the setting be `"f"`?